hail/python/hailtop/hailctl/dev/benchmark/cli.py

```diff
@@ -14,6 +14,8 @@ def main(args_):
                         type=int,
                         default=1,
                         help='Number of cores to use.')
+    parser.add_argument('-k', type=str, required=False, metavar='PATTERN', dest='pattern',
```

Comment:
why `-k`? If you give this a full name `--pattern` you won't need `dest`, and it'll be consistent with the rest of the code.

Reply:
consistency with pytest, I can add `--pattern` as well.

Reply:
ah, got it